core/locale/main/values/strings_ui.xml

```diff
@@ -308,11 +309,19 @@
     <string name="xapi_options_what">What</string>
     <string name="xapi_options_when">When</string>
     <string name="xapi_options_where">Where</string>
+    <string name="xapi_options_content">Content</string>
```

Comment:
We need to avoid string id prefixes so that we easily (and by default) use the same string in multiple parts of the app. We don't want to duplicate the same string (and create additional translation work). We should change 'xapi_options_what' to just 'what'. I think we will already have the word content from somewhere else in the app, so we don't need to add it.